picasso/src/test/java/com/squareup/picasso/DeferredRequestCreatorTest.java

```diff
@@ -54,48 +50,13 @@
     initMocks(this);
   }
 
-  @Config(reportSdk = 12)
-  @Test public void initAddsAttachListenerWhichDefersLayoutListener() {
```

Comment:
The other tests end with `Api11` which leads me to believe these are for API 12+. Those ones could be deleted though.